manifests/f/FlatspotSoftware/GloSC/2.0.6/FlatspotSoftware.GloSC.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: FlatspotSoftware.GloSC
+PackageVersion: 2.0.6
+PackageLocale: en-US
+Publisher: FlatspotSoftware

```

Comment:
```suggestion
Publisher: Peter Repukat - FlatspotSoftware
```
![image](https://user-images.githubusercontent.com/11600822/120535287-b7456f80-c3da-11eb-8047-0da2edbd4478.png)

Only the `PackageIdentifier` is fine now, and should still stay the same.